src/docs/reference/modules/Media.AmazonS3/README.md

```diff
@@ -154,6 +154,51 @@ The `BucketName` property and the `BasePath` property are the only templatable p
 }
 ```
 
+## Configuring a Local Emulator
+
+During development, instead of using an online S3 resource, you can use a local storage emulator too. This is especially important for development teams, since you don't want to step on each others' feet by using a shared storage.
```

Comment:
```suggestion
During development, instead of using an online S3 resource, you can use a local storage emulator too. This is especially important for development teams since you don't want to step on each others' feet by using a shared storage.
```